main/src/main/java/com/google/android/apps/muzei/settings/EffectsFragment.kt

```diff
@@ -189,15 +191,18 @@ class EffectsFragment : Fragment(R.layout.effects_fragment) {
             effectsLinked: Boolean = Prefs.getSharedPreferences(requireContext())
                     .getBoolean(Prefs.PREF_LINK_EFFECTS, false)
     ) {
-        val menuItem = binding.toolbar.menu.findItem(R.id.action_link_effects)
-        menuItem.setIcon(if (effectsLinked)
-            R.drawable.ic_action_link_effects
-        else
-            R.drawable.ic_action_link_effects_off)
-        menuItem.title = if (effectsLinked)
-            getString(R.string.action_link_effects)
-        else
-            getString(R.string.action_link_effects_off)
+        binding.toolbar.menu.findItem(R.id.action_link_effects).apply {
+            title = when (effectsLinked) {
+                true -> {
+                    setIcon(R.drawable.ic_action_link_effects)
+                    getString(R.string.action_link_effects)
+                }
+                false -> {
+                    setIcon(R.drawable.ic_action_link_effects_off)
+                    getString(R.string.action_link_effects_off)
+                }
+            }
```

Comment:
Switch this back to the previous setup:

1. `when (boolean)` can always be more succinctly written as an if statement
2. Don't `setIcon` as a side effect of setting the title. Keep this as two separate statements. Yes, the two if statements are intentional.